### PR TITLE
Varvara/feature/create user

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: naples/models/user.py [hunk 1]
```diff
@@ -88,3 +92,12 @@ def __repr__(self):
     def json(self):
         u = s.User.model_validate(self)
         return u.model_dump_json()
+
+    @classmethod
+    def get_user_by_email(cls, email: str, session: orm.Session | None = None) -> Self | None:
```
Comment (Taraslysak) [inline naples/models/user.py]: I think, session should not be optional argument here. Caller of this method should ensure that db session exists

File: naples/routes/auth.py [hunk 2]
```diff
@@ -58,3 +61,35 @@ def get_token(auth_data: s.Auth, db=Depends(get_db)):
...
+        password=data.password,
+    )
+    db.add(new_user)
+    db.commit()
+
+    log(log.INFO, "User [%s] signed up", new_user.email)
+
+    db_store = db.scalar(sa.select(m.Store).where(m.Store.user_id == new_user.id))
+
+    if not db_store:
```
Comment (Taraslysak) [inline naples/routes/auth.py]: Why do we need to make this db request? how can there be a possibility that this just created user has a store?

Comment (Taraslysak) [reply naples/routes/auth.py]: This scenario SHOULD NOT HAPPEN!!!! Any kind of this behaviour should be FIXED ASAP!

File: data/test_data.json [hunk 3]
```diff
@@ -79,15 +87,16 @@
       "longitude": "73.8648",
       "realtor_id": 1,
       "city_id": 4,
-      "amenities":["test_amenities1"],
+      "amenities": [
+        "test_amenities1"
+      ],
```
Comment (Taraslysak) [inline data/test_data.json]: Why do we need these amenities here?

Comment (Taraslysak) [reply data/test_data.json]: It does not look that they can be used in tests

**Review by Taraslysak — CHANGES_REQUESTED** [review]

File: data/test_data.json (hunk 3 above)
Comment (varvarashcherbyna7) [reply data/test_data.json]: I didn't add it, so I don't even know whether to delete it!

**Review by Taraslysak — APPROVED** [review]

**Pull request merged**